Dockerfile

```diff
@@ -2,14 +2,21 @@
 #
 # Usage:
 #
-# # Assemble the full dev environment. This is slow the first time.
-# docker build -t docker .
+# # Use make to build a development environment image and run it in a container.
+# # This is slow the first time.
+# make BIND_DIR=. shell
 #
-# # Mount your source in an interactive container for quick testing:
-# docker run -v `pwd`:/go/src/github.com/docker/docker --privileged -i -t docker bash
+# # Make a dockerd binary.
+# root@a8b2885ab900:/go/src/github.com/docker/docker# hack/make.sh binary
```

Comment:
Remove the prompt from this and the examples below

```
hack/make.sh binary
```

Reply:
Do you need to show that it is a root prompt, though?

Reply:
It should be inside the container, so it's going to be a root prompt. I think the important detail is to document that this is "inside the shell from the previous step".

Reply:
Done - although I removed the `root@` and retained the `#` ... can remove the `#` too but it looked OK to me.  Added a preceding note about the following commands being run from within the container.